helpers/helper-translation-discussion.php

```diff
@@ -185,6 +206,14 @@ public function get_css() {
 	public function get_js() {
 		return file_get_contents( $this->assets_dir . 'js/translation-discussion.js' );
 	}
+
+	public function sanitize_comment_topic( $comment_topic ) {
+		if ( ! in_array( $comment_topic, array( 'typo', 'context', 'question' ) ) ) {
+			$comment_topic = 'invalid topic';
```

Comment:
Nice! What about just falling back to a default like this?
```suggestion
			$comment_topic = 'question';
```

Reply:
I think there would be a problem if the commenter is on the English discussion page where there is no locale slug available in the url, then we won't know what locale to use.

The paragraph above is so because comments with topic set to 'questions' are posted under the language discussions which implies that the locale slug must be present.

Reply:
Good point. Then maybe let's just rather call it unknown? 
```suggestion
			$comment_topic = 'unknown';
```

Reply:
This works